.github/workflows/python_wheels.yml

```diff
@@ -316,7 +316,7 @@ jobs:
           docker run --rm -dit --name wheel-validation-examples wheel_validation:local
           status_sum=0
 
-          for ex in `find docs/sphinx/examples/python -name '*.py' -not -path '*/providers/*' -not -path '*/divisive_clustering_src/*'`; do
+          for ex in `find docs/sphinx/examples/python -name '*.py' -not -path '*/providers/*' -not -path '*/divisive_clustering_src/*' -not -path '*/utils_ipie.py' -not -path '*/vqe_cudaq_qnp.py'`; do
```

Comment:
Perhaps not for this PR, but we probably need to restructure some of these "support" files to put them all in a single location. They are causing our CI grief (see #2078), so perhaps they should all go into a directory called `docs/sphinx/examples/python/tutorials/support`, possibly with per-tutorial subdirectories for each one, just so we can keep them straight.